### PR TITLE
feat(generators): allow per-spec timeouts [skip-bc]

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/DI-3255 https://algolia.atlassian.net/browse/DI-3230

### Changes included:

- [x] allow api level default timeouts
- [ ] allow method level default timeouts (in a later ticket https://algolia.atlassian.net/browse/DI-3257)
- [x] update every specs

### CODE REVIEW

File: clients/algoliasearch-client-javascript/packages/client-common/src/transporter/createTransporter.ts [hunk 1]
```diff
@@ -142,6 +142,8 @@ export function createTransporter({
         responseTimeout: getTimeout(timeoutsCount, isRead ? timeout.read : timeout.write),
       };
 
+      console.log(payload);
```
Comment (millotp) [inline clients/algoliasearch-client-javascript/packages/client-common/src/transporter/createTransporter.ts:145]: #help-security
this guy is trying to leak PII

File: package.json [hunk 2]
```diff
@@ -31,16 +31,16 @@
   },
   "devDependencies": {
     "@openapitools/openapi-generator-cli": "2.15.3",
-    "@redocly/cli": "1.25.15",
+    "@redocly/cli": "1.25.13",
```
Comment (millotp) [inline package.json:34]: why the downgrade ?

**Review by millotp — COMMENTED** [review]
this is very cool

File: clients/algoliasearch-client-javascript/packages/client-common/src/transporter/createTransporter.ts (hunk 1 above)
Comment (shortcuts) [reply clients/algoliasearch-client-javascript/packages/client-common/src/transporter/createTransporter.ts:145]: what a detective you found my super evil plan!!!!!!

File: package.json (hunk 2 above)
Comment (shortcuts) [reply package.json:34]: ah that's a miss my bad

**Review by millotp — APPROVED** [review]

**Pull request merged**